terraform/hub-dev/main.tf

```diff
@@ -1,12 +1,16 @@
+variable "CALLYSTO_DOMAINNAME" {}
+variable "CALLYSTO_ZONE_ID" {}
```

Comment:
Yeah, probably. It works as expected with hub-dev. I can add it to templates if necessary as I see some hardcoded values there too
This would mean to replace those values with the variable names in locals {} definition.
Is this ok?